app/src/main/res/layout/fragment_main.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    >
+    <View
+        android:id="@+id/line"
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        android:visibility="gone"/>
+    <ImageView
+        android:id="@+id/qq_flag"
+        android:layout_width="@dimen/flag_width"
+        android:layout_height="@dimen/flag_height"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toLeftOf="@+id/ru_flag"
+        app:layout_constraintBottom_toTopOf="@+id/line"
+        android:src="@drawable/qq"
+        android:layout_marginBottom="24dp"
+        android:layout_marginLeft="@dimen/standard_margin"
+        android:layout_marginRight="@dimen/standard_margin" />
+    <TextView
+        android:id="@+id/qq_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/qaraqalpaqsha"
+        android:textSize="20sp"
+        android:textColor="@color/blue"
+        app:layout_constraintLeft_toLeftOf="@id/qq_flag"
+        app:layout_constraintRight_toRightOf="@+id/qq_flag"
+        app:layout_constraintTop_toBottomOf="@+id/qq_flag"/>
+    <ImageView
+        android:id="@+id/ru_flag"
+        android:layout_width="@dimen/flag_width"
+        android:layout_height="@dimen/flag_height"
+        android:layout_marginBottom="24dp"
```

Comment:
Move to dimens.xml, in general as a rule of thumb, try to set all dimensions in dimens.xml